### PR TITLE
Add benefits of embedding Google Map

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds  benefits of embedding Google Map

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/cms/page-builder-media-map.html

## Source 

- https://cloud.google.com/blog/products/maps-platform 
- https://cloud.google.com/blog/

### CODE REVIEW

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for reviewing!
I have implemented your suggestions, could you please approve this PR?
Thank you!

File: src/cms/page-builder-media-map.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ Use the _Map_ content type to add a map from [Google Maps Platform][1] to the [P
 
 To get the most from Google Maps Platform, you can customize the map, highlight your store location(s), and use Google [Places][2] to add rich information about your store to all Google maps.
 
+## Benefits of embedding Google Map
+
+1. Providing buyers with full contact information about the company (phone number, website, reviews, star ratings, etc.). By viewing the store address on Google Map, customers can also get other information right on your site.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-map.md]: Done ✅

File: src/cms/page-builder-media-map.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ Use the _Map_ content type to add a map from [Google Maps Platform][1] to the [P
...
+
+1. A Google map usually recommends and highlights nearby attractions, parks, restaurants, etc. This way the visitor can better determine your physical location in real life from nearby areas.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-map.md]: Done ✅

File: src/cms/page-builder-media-map.md [hunk 3]
```diff
@@ -6,6 +6,16 @@ Use the _Map_ content type to add a map from [Google Maps Platform][1] to the [P
...
+
+1. Facilitating the process of finding buyers at the addresses of your store. You don't have to open a new browser window at the same time while leaving the site to find your way. Buyers can directly use your sitemap
+
+1. If you have a chain of physical stores, getting a Google Map on your e-store is the fastest way to increase your brand awareness and credibility in the form of highlighted items.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/cms/page-builder-media-map.md]: Done ✅

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**